Error: <%= exception.message %>
Backtrace:
<% exception.backtrace.each do |bt| %>
<%= bt %>
<% end %>

